galsim/wcs.py

```diff
@@ -2188,35 +2317,60 @@ def world_origin(self): return self._world_origin
 
     def _u(self, x, y, color=None):
         if self._uses_color:
-            return self._ufunc(x,y,color)
+            try:
+                return self._ufunc(x,y,color)
+            except TypeError:
```

Comment:
Is this the only error that can be thrown here, or should we be catching something more general?

Reply:
It's the only kind of error I could think of for people writing code that works for floats but not numpy arrays.  (It's the same except statement that I had elsewhere before this PR.  I just moved it here in some code restructuring.)

Reply:
I went ahead and changed this to catching `Exception`, since it wouldn't be too surprising if I'm just insufficiently imaginative to think of a different exception that could be raised.  I switched these to follow the pattern in `RaDecFunction`.